pages/projects/[id].tsx

```diff
@@ -124,9 +125,20 @@ const Projects = (): JSX.Element => {
   });
 
   const loading = !Boolean(stories && Object.values(stories).length);
+  const { palette, type } = useTheme();
 
   return (
-    <div style={{ overflow: 'scroll' }}>
+    <div
+      style={{
+        overflow: 'scroll',
+        overflowX: 'scroll',
```

Comment:
(nit) We recently discovered that we should use `auto` instead of `scroll`. Setting the overflow to `scroll` will put the scrollbars into the container by default (even if nothing is overflowing). We don't see them on MAC because of the setting the OS has, but they'll be shown on some browsers on Windows computers. 

<img width="684" alt="Screen Shot 2020-10-08 at 3 40 57 PM" src="https://user-images.githubusercontent.com/42581689/95505640-b60f9000-097c-11eb-9b0f-5a693c6a2dc3.png">